scala_211/lift_basic/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -22,6 +22,7 @@ import net.liftmodules.JQueryModule
 class Boot {
   def boot {
     if (!DB.jndiJdbcConnAvailable_?) {
+      sys.props.put("h2.implicitRelativePath", "true")
```

Comment:
h2 1.4 now by default requires absolute paths. I'd originally set this in the build.sbt, but then thought that people will likely only require h2 for tests, so putting it alongside boot seemed appropriate. Is there a better place?